docs/quick-install.md

```diff
@@ -1,117 +1,90 @@
- - [Installation](#installation)
+The Smart Agent contains three main components:
 
+| Component | Description |
+|-----------|-------------|
+| Monitors  |  This component collects metrics from the host and applications. For a list of supported monitors and their configurations, see [Monitor Configuration](./monitor-config.md).            |
+| Observers |   This component collects metrics from services that are running in your environment. For a list of supported observers and their configurations, see [Observer Configuration](./observer-config.md).           |
+| Writer    |   This component collects metrics from configured monitors and then sends these metrics to SignalFx on a regular basis. If you are expecting your monitors to send large volumes of metrics through a single agent, then you must update the configurations. To learn more, see [Agent Configurations](./config-schema.md#writer).          |
 
-## Concepts
 
-The agent has three main components:
+## Review pre-installation requirements for the Smart Agent
 
-* _Monitors_ that collect metrics from the host and applications
+Before you download and install the Smart Agent on a **single** host, review the requirements below.
 
-* _Observers_ that discover applications and services running on the host
+(For other installation options, including bulk deployments, see [Advanced Installation Options](./advanced-install-options.md).)
 
-* a _Writer_ that sends the metrics collected by monitors to SignalFx
+| General requirements   |     Linux requirements      |  Windows requirements |
+|----------|:-------------:|------:|
+| <p>You must have access to your command line interface.</p> <p>You must uninstall or disable any previously installed collector agent from your host, such as collectd.</p>| <p>You must run kernel version 2.6 or higher for your Linux distribution.</p> <p>The Smart Agent is bundled with additional required dependencies, including a Java JRE runtime and a Python runtime. As a result, there is no need to proactively install additional dependencies.</p>| <p>You must run .Net Framework 3.5 on Windows 8 or higher.</p> <p>You must run Visual C++ Compiler for Python 2.7.</p>  |
 
 
-### Monitors
+## Install the Smart Agent
 
-Monitors collect metrics from the host system and services.  They are
-configured under the `monitors` list in the agent config.  For
-application specific monitors, you can define discovery rules in your monitor
-configuration. A separate monitor instance is created for each discovered
-instance of applications that match a discovery rule. See [Endpoint
-Discovery](./auto-discovery.md) for more information.
+### Step 1. Install the SignalFx Smart Agent on Single Host
 
-Many of the monitors are built around [collectd](https://collectd.org), an open
-source third-party monitor, and use it to collect metrics. Some other monitors
-do not use collectd. However, either type is configured in the same way.
+<details>
+<summary>Linux</summary>
+<br>
```

Comment:
I like this Summary thing! Never saw it before.

Reply:
But all I see when I expand Linus is the sh statement. So I'm guessing headings aren't supported. So you need to rework this and review it live, and then I'll look again. 

Also I'm not sure "Optional 1" and "Optional 2" work as headings, even if they were visible. Maybe use Summary for those too?

Reply:
This content still isn't showing up in the build. Maybe we should discuss this PR in our 1x1 tomorrow.

![image](https://user-images.githubusercontent.com/17709550/70097529-78b0d800-15de-11ea-8cef-3a9eb553532e.png)

Reply:
Hmm bizarre; I see it working fine within GitHub. 
<img width="960" alt="Screen Shot 2019-12-03 at 7 08 27 PM" src="https://user-images.githubusercontent.com/54906773/70100651-59c33d80-1600-11ea-92b4-7cc0d36615da.png">